composer.json

```diff
@@ -21,11 +21,11 @@
         }
     },
     "require": {
-        "php": ">= 7.0",
+        "php": ">= 8.0",
```

Comment:
We talked about this IRL, but if we can leave PHP 7 as an option for now to account for the existing sites we have that run that version, that would be ideal. Once WP's PHP 8+ support is out of beta, we'll probably want to revisit this and other dependencies to require PHP 8.